doc/classes/TabContainer.xml

```diff
@@ -42,14 +42,14 @@
 			<return type="int">
 			</return>
 			<description>
-				Return the previous tab index that was being shown.
+				Returns the previous tab index that was being shown.
 			</description>
 		</method>
 		<method name="get_tab_align" qualifiers="const">
 			<return type="int" enum="TabContainer.TabAlign">
 			</return>
 			<description>
-				Return tab alignment, from the ALIGN_* enum.
+				Returns the tab alignment, from the ALIGN_* enum.
```

Comment:
Change the , to a . and make the last sentence conform to the enum standard: See the ALIGN_* constants.